packages/odata-common/src/properties-util.spec.ts

```diff
@@ -1,14 +1,24 @@
-import { commonEntityApi } from '@sap-cloud-sdk/test-services-odata-common/common-entity';
-import { isNavigationProperty } from './properties-util';
+import {
+  CommonEntity,
+  commonEntityApi
+} from '@sap-cloud-sdk/test-services-odata-common/common-entity';
+import { nonEnumerable, isNavigationProperty } from './properties-util';
 
 describe('isNavigationProperty', () => {
+  it('returns false for non-enumerable property', () => {
+    nonEnumerable(CommonEntity, '_entityName');
```

Comment:
[req] this test should not be under the `isNavigationProperty` description.

Reply:
I made another description for the test